docs/source/el7toel8/actor-rhel7-to-rhel8.md

```diff
@@ -327,6 +327,17 @@ data is stored in the database for the current session:
 ```shell
 snactor run --save-output IPUWorkflowConfig
 ```
+Since the leapp supports several upgrade paths, the `snactor` needs to know target system and upgrade flavor specification for correct execution.
```

Comment:
```suggestion
Since the leapp upgrade repositories support several upgrade paths, the `snactor` needs to get additional data for the correct execution, like the release of the target system, the flavor specification, etc.
```